locustfile.py

```diff
@@ -0,0 +1,205 @@
+import os
+import logging
+from locust import HttpUser, task, between
+from utils.env_helper import load_env_variables
+
+user_count = 0
+
+
+class QuickstartUser(HttpUser):
+    global user_count
+    wait_time = between(1, 5)
+
+    """
+    all_api
+
+    Sends queries to all cupboard apis
+    """
```

Comment:
Put the block comment inside the function (right under the `def all_api(self)`). Do this for the other block comment too.